test/integration/KarmaTestRunner.it.ts

```diff
@@ -13,7 +13,19 @@ describe('KarmaTestRunner', function () {
   let expectToHaveFailedTests = (result: RunResult, expectedFailureMessages: string[]) => {
     const actualFailedTests = result.tests.filter(t => t.status === TestStatus.Failed);
     expect(actualFailedTests).to.have.length(expectedFailureMessages.length);
-    actualFailedTests.forEach(failedTest => expect(failedTest.failureMessages[0]).to.contain(expectedFailureMessages.shift()));

```

Comment:
I agree! I forgot about the possiblity of just casting it to `any` for a second there.. I've fixed it!